client/src/pages/guide/english/css/css3-media-queries/index.md

```diff
@@ -50,8 +50,39 @@ The `p` tag will have a padding of 30px as soon as the screen reaches min 768px
 
 The `p` tag will have a padding of 30px as soon as the screen reaches min 768px width or its resolution reaches min 150dpi.
 
+### And beyond!
+
+There are more operators beyond the main two, a full list with examples can be found [https://css-tricks.com/logic-in-media-queries/](in this article from CSS Tricks.)
+
+Beyond the core uses of media queries for mobile-first web design shown above, media queries can do a lot, especially for web accessibility. Here are just a few examples:
+
+1. Adjusting for screen readers that convert website text to speech for the visually impaired (for example, ignoring non-essential text)
+```css
+    @media speech {
+      /* ... */
+    }
+```
+2. Allowing for more graceful zooming in for the minorly visually impaired, such as many elderly people
+3. Allowing smoother experiences for those who prefer or need less animation to read a page.
+```css
+    @media (prefers-reduced-motion: reduce) {
+      .animation {
+        animation: none;
+        -webkit-animation: none;
+      }
+    }
+```
+4. Restyling a page for when it's printed as opposed to read on a screen
+```css
+    @media print {
+      /* ... */
+    }
+```
+
 ### More Information
 * [MDN - media queries](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries)
 * [W3 Schools - @media rule](https://www.w3schools.com/cssref/css3_pr_mediaquery.asp)
 * [CSS Tricks Standard Device Widths Atricle](https://css-tricks.com/snippets/css/media-queries-for-standard-devices/)
 * [https://alistapart.com/article/responsive-web-design](Ethan Marcotte A List Apart Atricle on  Responsive Web Design)
+* [http://bradfrost.com/blog/post/7-habits-of-highly-effective-media-queries/](Brad Frost 7 habits of highly effective media queries)
```

Comment:
The links are not formatted correctly, the text should be in the square bracket and the link should be in the parentheses. Can you fix it?

Reply:
Ok, I'll get right on it.